Source/DafnyCore/Compilers/Compiler-js.cs

```diff
@@ -2497,7 +2491,7 @@ protected class ClassWriter : IClassWriter {
         }
         nodeProcess.StandardInput.Flush();
         nodeProcess.StandardInput.Close();
-        return 0 == RunProcess(nodeProcess, outputWriter);
+        return 0 == WaitForExit(nodeProcess, outputWriter);
```

Comment:
Why does Javascript use WaitForExit and not the other compilers?

Reply:
Because it starts the process itself, whereas for the other ones the process is started by `RunProcess`